### ISSUE TITLE
Featured image not getting displayed until we update the post

### ISSUE THREAD

**Issue by zumich** [weDevsOfficial/wp-user-frontend#408]
Hi, I have a form here https://wordpress-158272-475719.cloudwaysapps.com/add-new-event/
The featured image that I've added does not get displayed until I update the post itself from the backend.

**Comment by zumich** [comment]
CHeck the form here https://compi.in/add-new-event/

I've tested for plugin and theme incompatibility, but that doesn't seem to be the issue

**Comment by itowhid06** [comment]
Hello @zumich , 

Are you still facing the issue? 
I've just created a post with the form link you provided and the featured image is showing as it is intended. post link: [](https://compi.in/event/post_title-2/)

Best regards

**Comment by zumich** [comment]
Nah, I've fixed it.  It seems some of the css applied to the image was a problem. I used `wp_get_attachment_url( get_post_thumbnail_id($post->ID)) `and commented out the original code and that fixed it. This only works when you have to display just the featured image and not the gallery
